modules/compliance-tailored-profiles.adoc

```diff
@@ -74,6 +77,24 @@ spec:
 |`setValues`
 | A list of name, rationale, and value groupings. Each name refers to a name of the value set. The rationale is human-readable text describing the set. The value is the actual setting.
 |===
++
+.. Add the `tailoredProfile.spec.manualRules` attribute :
++
+.Example `tailoredProfile.spec.manualRules.yaml`
+[source,yaml]
+----
+apiVersion: compliance.openshift.io/v1alpha1
+kind: TailoredProfile
+metadata:
+  name: rhcos4-manual-scc-check
```

Comment:
Sorry, this is on me, because I adapted this example from another one and forgot to change the name..we should change the prefix from `rhcos4-` to `ocp4-` for consistency's sake.

Reply:
Thank you! I have made the updates! :)